GoogleForCreators/web-stories-wp
# Templates: Update Fashion Template Aspect Ratio to 9:16

## Summary
Updates Fashion template to new aspect ratio and fixes other critiques put in latest template design pass for fashion.

Designs & Critiques:
https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=766%3A61

## Relevant Technical Choices
N/A

## To-do
N/A

## User-facing changes
Fashion template design update.

## Testing Instructions
Go to explore templates and view the fashion template. Each page should now match these designs:
https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=766%3A61

---



Fixes #2471

## Screenshots
<img width="1440" alt="Screen Shot 2020-06-17 at 1 55 08 PM" src="https://user-images.githubusercontent.com/35983235/84944362-def4d800-b0a2-11ea-88f3-945c93c7077f.png">
